contrib/systemd/ocid.service

```diff
@@ -8,7 +8,7 @@ Type=notify
 EnvironmentFile=-/etc/sysconfig/ocid-storage
 EnvironmentFile=-/etc/sysconfig/ocid-network
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/bin/ocid \
+ExecStart=/usr/local/bin/ocid \
```

Comment:
For a sample systemd service, I think we should retain /usr/bin/ocid as that's where the distributions will package / install it at.